unit_test/graph/Test_Graph_coarsen.hpp

```diff
@@ -272,6 +272,7 @@ crsMat gen_grid() {
   return A;
 }
 
+#if !defined(KOKKOS_ENABLE_CUDA) || defined(KOKKOS_ENABLE_CUDA_LAMBDA)
```

Comment:
Oops, missed this on the original review.

Since all of the internal test functions in this file rely on lambdas, why not put the if/endif guards around the whole file? That would also keep no-op tests from appearing in the gtest list.

Reply:
Yeah, I could do that direclty in `Test_Graph.hpp`, that's a good point

Reply:
What worries me is not that you forgot but rather that the auto-tester seems to have forgotten to run the CUDA checks...

Reply:
true... I couldn't remember if those builds were supposed to have Kokkos_ENABLE_CUDA_LAMBDA on or off.